packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -93,51 +121,31 @@ function NavigationMenuItemEdit( {
 						name="link"
 						icon="admin-links"
 						title={ __( 'Link' ) }
-						onClick={ () => setIsLinkOpen( ! isLinkOpen ) }
+						onClick={ noop }
```

Comment:
It seems like a deviation from our normal implementation, having `onClick` as a `noop`, while `onMouseDown` is defined instead. It'd be great to add a comment about why this is so that someone like me doesn't come a long later and change it 😄

Reply:
Fair enough. It sounds like a very valid reason to doc :-D